src/index.js

```diff
@@ -105,6 +115,41 @@ export default class VueI18n {
     })
   }
 
+  _checkLocaleMessage (locale: Locale, level: WarnHtmlInMessageLevel, message: LocaleMessageObject): void {
+    const stack: Array<string> = []
+
+    const fn = (message: mixed, stack: Array<string>) => {
+      if (isPlainObject(message)) {
+        Object.keys(message).forEach(key => {
+          const val = message[key]
+          const hasObject = isPlainObject(val)
+          stack.push(key)
+          hasObject && stack.push('.')
+          fn(val, stack)
+          hasObject && stack.pop()
+        })
+      } else if (Array.isArray(message)) {
+        message.forEach((item, index) => {
+          stack.push(`[${index}]`)
+          fn(item, stack)
+          stack.pop()
+        })
+      } else if (typeof message === 'string') {
+        const ret = htmlTagMatcher.test(message)
+        if (ret) {
+          const msg = `Detect unsafe locale message '${message}' of keypath '${stack.join('')}' at '${locale}', suggest use component interpolation with '<i18n>'`
```

Comment:
```suggestion
          const msg = `Detected HTML in message '${message}' of keypath '${stack.join('')}' at '${locale}'. Consider  component interpolation with '<i18n>' to avoid XSS. See https://kazupon.github.io/vue-i18n/guide/interpolation.html`
```

URL could be shortend by https://bitly.com/

Reply:
Thanks a lot!